### PR TITLE
feat: add get started and use helvetica font

### LINKED ISSUE #15: Add the Get Started Section and use helvetica font

### PR DESCRIPTION
**Pull request by hasanagh**
adds a getting started page and uses helvetica font

closes #15

### CODE REVIEW

File: src/styles/layout.css [hunk 1]
```diff
@@ -845,7 +854,7 @@ footer .right-cols .columns p {
 }
 
 footer h3 {
-  font: 600 13px/24px 'Open Sans', sans-serif;
+  font: 600 13px/24px 'helvetica', sans-serif;
```
Comment (juancarlosfarah) [inline src/styles/layout.css]: Helvetica with caps.

File: src/styles/layout.css [hunk 2]
```diff
@@ -580,7 +589,7 @@ ul.features li:nth-child(even) {
   margin: 0 auto;
 }
 #testimonials h1 {
-  font: bold 16px/24px 'Montserrat', sans-serif;
+  font: bold 16px/24px 'helvetica', sans-serif;
```
Comment (juancarlosfarah) [inline src/styles/layout.css]: Uppercase.

File: src/styles/layout.css [hunk 3]
```diff
@@ -213,7 +222,7 @@ ul.nav li.current button {
   margin: 54px auto 24px auto;
 }
 .hero-text h1 {
-  font: 600 40px/1.2em 'Montserrat', sans-serif;
+  font: 600 40px/1.2em 'helvetica', sans-serif;
```
Comment (juancarlosfarah) [inline src/styles/layout.css]: Uppercase, for consistency.

Comment (juancarlosfarah) [reply src/styles/layout.css]: There are multiple occurrences of this typo below.

File: src/styles/layout.css [hunk 4]
```diff
@@ -118,6 +118,15 @@ header .header-social li a:hover {
 
 /* primary navigation
 --------------------------------------------------------------------- */
+
+.NavLink {
```
Comment (juancarlosfarah) [inline src/styles/layout.css]: Was this not done and merged already?

File: src/components/navmenu.js [hunk 5]
```diff
@@ -37,21 +37,13 @@ export default function () {
...
-              >
-                {value.label}
-              </button>
-            </li>
-          );
-        })}
+        <li key="0">
+          <button type="button">
+            <Link to="/get-started" className="NavLink">
+              GET STARTED
```
Comment (juancarlosfarah) [inline src/components/navmenu.js]: I think you don't need to write this in caps.

File: src/components/header.js [hunk 6]
```diff
@@ -8,14 +8,14 @@ import HeaderSocial from './header-social';
 export default () => (
   <header>
     <div className="logo">
-      <AnchorLink to="/#top">
+      <Link to="/">
```
Comment (juancarlosfarah) [inline src/components/header.js]: Add `alt="Logo"` or `alt="Graasp"`.

File: content/get-started.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+headline: Download our getting started guides in both English and French
+
+body: Download the open source, community version of Graasp Desktop for your preferred operating system.
+
+button1:
+  label: Get Started (ER)
```
Comment (juancarlosfarah) [inline content/get-started.yaml]: Typo: `EN`.

File: content/get-started.yaml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+headline: Download our getting started guides in both English and French
+
+body: Download the open source, community version of Graasp Desktop for your preferred operating system.
```
Comment (juancarlosfarah) [inline content/get-started.yaml]: Don't we need to change this?

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Hi Hassan. Thanks for this. Just some minor changes throughout. Could you also include a screenshot in the PR? Thanks!

File: src/components/navmenu.js (hunk 5 above)
Comment (hasanagh) [reply src/components/navmenu.js]: So all letters without caps, or only the "G" and the "S" ?

File: content/get-started.yaml (hunk 8 above)
Comment (hasanagh) [reply content/get-started.yaml]: I am not using it, will remove

File: src/styles/layout.css (hunk 4 above)
Comment (hasanagh) [reply src/styles/layout.css]: No this is done to show the get started page now in the navigation bar instead of the home page sections

**Comment by hasanagh** [comment]
Screen Shot 2020-09-28 at 8 14 11 PM

File: src/styles/layout.css (hunk 4 above)
Comment (juancarlosfarah) [reply src/styles/layout.css]: OK

File: src/components/navmenu.js (hunk 5 above)
Comment (juancarlosfarah) [reply src/components/navmenu.js]: Get Started

File: content/get-started.yaml (hunk 8 above)
Comment (juancarlosfarah) [reply content/get-started.yaml]: OK. For the headline, let's make it shorter and capitalised: "Download Our Getting Started Guide".

File: src/components/navmenu.js (hunk 5 above)
Comment (hasanagh) [reply src/components/navmenu.js]: Are you sure about that? I don't know why I feel the full caps version is more appealing to my eyes at least.  😆 

Screen Shot 2020-09-28 at 8 57 58 PM

File: content/get-started.yaml [hunk 9]
```diff
@@ -0,0 +1,11 @@
+headline: Download our getting started guides in both English and French
```
Comment (juancarlosfarah) [inline content/get-started.yaml]: "Download Our Getting Started Guide"

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Just a minor change in the headline message.

**Review by juancarlosfarah — APPROVED** [review]

**Pull request merged**